### ISSUE TITLE
PIN does not work after restoring a backup made on another phone

### ISSUE THREAD

**Issue by orium** [bitcoin-wallet/bitcoin-wallet#412]
Hi,
I recently acquired an new phone and use the backup/restore method to transfer my wallet from the old phone to the new one.  The wallet was successfully imported but I was unable to spend because the PIN was not recognized as valid.  The exact same pin on the old phone worked (and I ended up transfering the coins from the old phone to the new one by creating a new wallet on the new phone and doing a normat bitcoin transaction to it).

I will be able to provide more details, including encrypting wallet backups from the old phone, but meanwhile can this have an "obvious" explanation (like selecting the iteration number based on current hardware instead of reading it from the backup file)?

**Comment by schildbach** [comment]
Did you change the spending PIN between when the backup was created and restored on the new phone?

**Comment by orium** [comment]
I'm confident I didn't change it.  I will try to reproduce the problem.

**Comment by schildbach** [comment]
Can you use Options > Report issue and send your wallet dump, on both phones the old one and the new one (with your wallet restored)? Thanks.

**Comment by orium** [comment]
I tried to reproduce this with no success.  I have a big table now of wallet-backup files (some a few years old) and I got consistent results with both the old phone and the new phone.  Newly made backups also work fine.

I can't really explain how but it seems more likely that this was my fault.  (The only explanation I have is for me to change the PIN on the new phone after restoring the backup, but I think I would remember that.)  Anyway, given this I will close the issue.
